corehq/apps/sms/models.py

```diff
@@ -775,6 +777,67 @@ class Meta:
         app_label = "sms"
 
 
+class CommConnectCaseSQL(CommCareCaseSQL, CommCareMobileContactMixin):
+    class Meta:
+        app_label = "sms"
```

Comment:
Spent some time debugging the tests and figured out that it's because this needs to be `form_processor` in order for the `PartitionRouter` (database router for sharded setup) to work.